### PR TITLE
Making Util.splitMessage always return an array

### PR DESCRIPTION
**Pull request by TNThacker2015**
Util.splitMessage sometimes returns an array, but other times it returns a string. This should make it so that it always returns an array.

**Please describe the changes this PR makes and why it should be merged:**
This would make Util.splitMessage always return an array.
It should be merged because Util.splitMessage would be more consistent

**Status**
- [ ] Code changes have been tested against the Discord API, or there are no code changes
- [ ] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [x] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/util/Util.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ class Util {
    * @returns {string|string[]}
```
Comment (SpaceEEC) [inline src/util/Util.js]: As this now only return an array:
```suggestion
    * @returns {string[]}
```

**Comment by Gawdl3y** [comment]
Not really a fan personally, since the goal was intentionally to not modify the input whatsoever unless it actually needed to be split up.

File: src/util/Util.js [hunk 2]
```diff
@@ -53,10 +53,10 @@ class Util {
    * Splits a string into multiple chunks at a designated character that do not exceed a specific length.
    * @param {string} text Content to split
    * @param {SplitOptions} [options] Options controlling the behavior of the split
-   * @returns {string|string[]}
+    * @returns {string[]}
```
Comment (kyranet) [inline src/util/Util.js]: Looks like a space was added accidentally.
```suggestion
   * @returns {string[]}
```

**Comment by TomSputz** [comment]
https://github.com/discordjs/discord.js/blob/master/typings/index.d.ts#L1102

This will need to be updated

**Pull request merged**